salttesting/case.py

```diff
@@ -321,7 +321,7 @@ def detach_from_parent_group():
                     break
         tmp_file.seek(0)
 
-        if sys.version_info.major > 2:
+        if sys.version_info > (2,):
```

Comment:
I think my first comment misled you in the wrong direction. We want to make sure we're running python 3 on that `if` check.